src/core/book-keeper.js

```diff
@@ -1,33 +1,34 @@
 'use strict';
 
 var events = require('./event/event');
+var interval = require('../../config/default');
 
 class BookKeeper {
     constructor (eventBus) {
-        eventBus.emitEvent(events.FILL_REMINDER_QUEUE, {
-            cb: function () {
-                setInterval(function () {
-                    eventBus.emitEvent(events.FETCH_NEXT_REMINDER, {
-                        cb: function (err, data) {
-                            if(err) {
-                                console.err('Problem with events.FETCH_NEXT_REMINDER: ' + err);
-                            }
-
-                            var date = new Date();
-                            var now = date.getTime();
-                            // add field 'itemType' to JSON data object
-                            if(data.time < now) {
-                                data['itemType'] = 'reminder';
-                                eventBus.emitJiboEvent('INTERRUPT_IDLE', data);
-                            } else {
-                                // do nothing
-                            }
-                        }
-                    });
-                }, 1000);
-            }
+
+        eventBus.emitEvent(events.FILL_REMINDER_QUEUE, { }, function () {
+
         });
+
+        setInterval(function () {
+            eventBus.emitEvent(events.FETCH_NEXT_REMINDER, {}, function (err, data) {
+                if(err) {
+                    console.err('Problem with events.FETCH_NEXT_REMINDER: ' + err);
+                    return;
+                }
+
+                var date = new Date();
+                var now = date.getTime();
+
+                // add field 'itemType' to JSON data object
+                if(data.time < now) {
+                    data['itemType'] = 'reminder';
```

Comment:
Is better style to write `data.itemType` instead since it's a string literal (lint should've also caught this too I think, can you verify this?)